framework/include/utils/JvarMapInterface.h

```diff
@@ -89,6 +91,10 @@ class JvarMapInterfaceBase : public T
    */
   bool mapJvarToCvar(unsigned int jvar, unsigned int & cvar);
 
+protected:
+  /// number of coupled moose variables
+  const unsigned int _n_args;
```

Comment:
This would be a great opportunity to name this something other than `_n_args` in preparation for changing the name of the `args` parameter...

I'll leave it up to you if you would rather go back and change the code in all the classes that use this down the road. I imagine that changing this would result in API breakage in apps that depend on this, so it's probably better to do it now.

We could just settle on an appropriate replacement name for args now to move forward with this.  Suggestions: `variable_arguments`, `variable_dependencies`, `dependent_variables`, `coupled_variables`. I think my favorite is `variable_dependencies`, in which case this could be `_n_var_deps`.

Reply:
Ok, I'm going ahead with `variable_dependencies` and `_n_var_deps`.

Reply:
Argh, the problem is that for parsed materials we also use `args` to list the arguments of the function. `variable_dependencies` would not be a fitting name here. I just search/replaced over 600 parameters in input files breaking all parsed materials. I we change to variable dependencies the replacing will be a lot of work. I' would like to table that and not do it in this PR (or ever for that matter, but that's a different discussion).